src/transformers/babel/index.ts

```diff
@@ -6,10 +6,13 @@ import { getTransformerFactory } from '../../core/transformers/getTransformerFac
 import { libraryName } from '../../constants/libraryName';
 import { ProgramRepository } from '../../core/program/ProgramRepository';
 
-logLogo();
-
 export default function(api: any, options?: IDiConfig) {
     initDiConfig(options);
+
+    if (!options?.disableLogoPrint) {
+        logLogo();
```

Comment:
This function will be called for each file, so in the result, if the logo is not disabled, we will log logo a lot of times